infra/cifuzz/cifuzz.py

```diff
@@ -103,41 +117,43 @@ def build_fuzzers(project_name, project_repo_name, commit_sha, git_workspace,
       '-c',
   ])
   command.append(bash_command)
-
   if helper.docker_run(command):
     logging.error('Building fuzzers failed.')
     return False
   return True
 
 
-def run_fuzzers(project_name, fuzz_seconds, out_dir):
+def run_fuzzers(project_name, fuzz_seconds, workspace):
   """Runs all fuzzers for a specific OSS-Fuzz project.
 
   Args:
     project_name: The name of the OSS-Fuzz project being built.
     fuzz_seconds: The total time allotted for fuzzing.
-    out_dir: The location in the shared volume to store output artifacts.
+    workspace: The location in a shared volume to store a git repo and build
+      artifacts.
 
   Returns:
     (True if run was successful, True if bug was found).
   """
-  if not out_dir or not os.path.exists(out_dir):
-    logging.error('Unreachable out_dir argument %s.', format(out_dir))
+  # Validate inputs.
+  if not os.path.exists(workspace):
```

Comment:
is it necessary to check this in both `build_fuzzers` and `run_fuzzers? also, the error message is different. I'd recommend verifying this on the caller side in a single place

Reply:
I wanted to keep these two functions as self contained as possible. But we should discuss the costs and benefits of doing this.